src/libs/actions/Report.js

```diff
@@ -324,15 +324,23 @@ function subscribeToReportTypingEvents(reportID) {
     // Typing status is an object with the shape {[login]: Boolean} (e.g. {yuwen@expensify.com: true}), where the value
     // is whether the user with that login is typing on the report or not.
     Pusher.subscribe(pusherChannelName, 'client-userIsTyping', (typingStatus) => {
-        const login = _.first(_.keys(typingStatus));
+        let userObject = typingStatus;
+        let login = _.first(_.keys(typingStatus));
         if (!login) {
             return;
         }
 
+        // If the user is typing on expensify.com (like concierge) the format will actually be [userLogin: Concierge]
+        // We need to make sure to handle that case or it will say "userLogin is typing..."
+        if (login === 'userLogin') {
+            login = typingStatus.userLogin;
+            userObject = {[typingStatus.userLogin]: true};
+        }
```

Comment:
If we are going in this direction... it feels like it should be some kind of utility method to me that takes the `typingStatus` object and converts it over to the correct format something like...

`getNormalizedTypingStatus()`

that way we can clearly see the different shapes this data might take and how we handle it differently and why.

Reply:
Love it. I'll hop on it in the morning

Reply:
Updated. Thanks for the suggestion